src/event.cairo

```diff
@@ -47,6 +50,20 @@ mod StarkPassEvent {
         amount: u256,
     }
 
+    #[derive(Drop, starknet::Event)]
```

Comment:
what does this decorator do?

Reply:
@muchiNoChi Not sure about the `Drop` but the starknet::Event basically means that the structure extends the Event, so it can be used as an event